.github/workflows/e2e_cloud.yml

```diff
@@ -0,0 +1,94 @@
+name: E2E Cloud
+
+concurrency:
+  group: ci-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - staging
+      - trying
+  pull_request:
+    branches: [main]
+    paths:
+      - "**"
+      - "!/*.md"
+      - "!/**.md"
```

Comment:
This should run on schedule instead of on pushes and pull_requests, is ok to keep this until we have a green check tho

Reply:
Agree!